jme3-terrain/src/main/java/com/jme3/terrain/geomipmap/MultiTerrainLodControl.java

```diff
@@ -48,21 +51,41 @@
  * @author Brent Owens
  */
 public class MultiTerrainLodControl extends TerrainLodControl {
-    
-    List<TerrainQuad> terrains = new ArrayList<TerrainQuad>();
-    private List<TerrainQuad> addedTerrains = new ArrayList<TerrainQuad>();
-    private List<TerrainQuad> removedTerrains = new ArrayList<TerrainQuad>();
 
-    public MultiTerrainLodControl(List<Camera> cameras) {
-        this.cameras = cameras;
-        lodCalculator = new DistanceLodCalculator(65, 2.7f);
+    private SafeArrayList<TerrainQuad> terrains;
```

Comment:
because we have foreach operation with the collection for each call of the method prepareTerrain().